src/data/text/move_names.h

```diff
@@ -3,15 +3,15 @@ const u8 gMoveNames[MOVES_COUNT][MOVE_NAME_LENGTH + 1] =
     [MOVE_NONE] = _("-"),
     [MOVE_POUND] = _("Pound"),
     [MOVE_KARATE_CHOP] = _("Karate Chop"),
-    [MOVE_DOUBLE_SLAP] = _("Doubleslap"),
+    [MOVE_DOUBLE_SLAP] = _("DoubleSlap"),
```

Comment:
I think some move names can have spaces in between the words and still fit in the 12 character limit like DoubleSlap and therefore match up to the Gen 7 names. Might be good to look into that.